ports/gamenetworkingsockets/portfile.cmake

```
@@ -1,23 +1,48 @@
-vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ValveSoftware/GameNetworkingSockets
-    REF 9875f39ad3fae8c913daa310c7ff0a7a54b1e6c2 # v1.3.0
-    SHA512 edf814ef10a11b67045d90d589a3bd6471fa46bc3f16cf1e83d1806df6bb48306dcda4096a2ff80fee78b24ce36d164cd46808680c9f5f8f546455fce7bf9aa0
+    REF 5c793b9f1a507aa9670c8634817bde48b744428b # v1.3.0
+    SHA512 6ed8edbfe0b899a1e0aaba4cc7cae2e101fb292b18becb29b0da47c0894d603cad5c4b0983caddbcf269537a2022bd2cc965a19530615a0aa68406278e127525
     HEAD_REF master
 )
 
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+        "webrtc"    USE_STEAMWEBRTC
+)
+
+vcpkg_from_git(
+    OUT_SOURCE_PATH WEBRTC_SOURCE_PATH
+    URL https://webrtc.googlesource.com/src
+    REF 30a3e787948dd6cdd541773101d664b85eb332a6
+    HEAD_REF MAIN
+)

```

```suggestion
if (USE_STEAMWEBRTC)
    vcpkg_from_git(
        OUT_SOURCE_PATH WEBRTC_SOURCE_PATH
        URL https://webrtc.googlesource.com/src
        REF 30a3e787948dd6cdd541773101d664b85eb332a6
        HEAD_REF MAIN
    )
endif()
```
This should only be executed when the user select the webrtc feature.